src/main/kotlin/net/corda/training/flow/IOUSettleFlow.kt

```diff
@@ -45,16 +46,15 @@ class IOUSettleFlow(val linearId: UniqueIdentifier, val amount: Amount<Currency>
 
         // Step 4. Check we have enough cash to settle the requested amount.
         val cashBalance = serviceHub.getCashBalance(amount.token)
-        if (cashBalance == null) {
-            throw IllegalArgumentException("Borrower has no ${amount.token} to settle.")
```

Comment:
There is a unit test which depends on this specific  message which will now likely fail: borrowerMustHaveCashInRightCurrency

Reply:
It can't be null as cashBalance is not nullable. So null check was redundant.

Reply:
That is true but make sure the removal doesn't affect the associated test: https://github.com/roger3cev/corda-training-solutions/blob/master/src/test/kotlin/net/corda/training/flow/IOUSettleFlowTests.kt#L158

Reply:
Yeps, all looks good.